docs/installation/centos.md

```diff
@@ -5,18 +5,15 @@ description = "Instructions for installing Docker on CentOS"
 keywords = ["Docker, Docker documentation, requirements, linux, centos, epel, docker.io,  docker-io"]
 [menu.main]
 parent = "smn_linux"
+weight=-4
 +++
 <![end-metadata]-->
 
 # CentOS
 
-Docker is supported on the following versions of CentOS:
-
-* CentOS 7.X 
-
-Installation on other binary compatible EL7 distributions such as Scientific
-Linux might succeed, but Docker does not test or support Docker on these
-distributions.
+Docker is supported CentOS 7.X. An installation of other binary compatible EL7
```

Comment:
An installation of -> An installation on

Do we need the "binary compatible"? I think it complicates the sentence :-)